views/docs.ejs

```diff
@@ -1,37 +1,47 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= title %></title>
-    <!--<link rel='stylesheet' href='/stylesheets/style.css' />-->
-    <link rel='stylesheet' href='/stylesheets/redoc/main.css' />
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- Place this tag in your head or just before your close body tag. -->
-    <script async defer src="https://buttons.github.io/buttons.js"></script>
-    
+	<title><%= title %></title>
+	<link rel='stylesheet' href='/stylesheets/redoc/main.css' />
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<!-- Place this tag in your head or just before your close body tag. -->
+	<script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
what does this script do?  External sources should be loaded through bower so we can fix the version numbers used when rolled out into production.  When we develop against github's script and they make a change it can affect our application adversely.